### ISSUE TITLE
MacOS Monterey beta OS swallowing keystrokes

### ISSUE THREAD

**Issue by Nutlope** [withfig/fig#335]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] My issue does not relate to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If it does, please create your issue here: [withfig/autocomplete](https://github.com/withfig/autocomplete/issues?q=)

### Steps to Reproduce
Use Monterey beta OS swallowing keystrokes

### Expected Results
Autocomplete should work well.

### Actual Results
Swallowing a surprisingly large amount of keystrokes, especially with CMD+Tab on Mac, and after getting focus.

### Environment

```bash
N/A
```

### Is this issue related to the docs?

- [ ] Yes
- [X] No

**Comment by mschrage** [comment]
@Nutlope Would it be possible to get any more details or, ideally, a screen recording demonstrating the behavior?

**Comment by Nutlope** [comment]
> @Nutlope Would it be possible to get any more details or, ideally, a screen recording demonstrating the behavior?

Brendan to follow up with him as discussed. It's the founder of utopia that was having this issue

**Comment by fwesss** [comment]
@mschrage @Nutlope I've come across when the autocomplete.onlyShowOnTab setting is enabled. Using cmd + tab will not work from a terminal where Fig is active due to the tab key being swallowed.

**Comment by GuilleDF** [comment]
Hi all! Any news on this issue? I'm having the same behavior in MacOS Big Sur. I'm happy to provide a reproduction video if needed 😃

**Comment by mschrage** [comment]
Hey @GuilleDF! A screen recording would be super helpful! 

I think this may be fixed in the next release (v1.0.49) depending on which keys were being intercepted for you?

**Comment by GuilleDF** [comment]
Hi @mschrage! Here's the repro video 😄 
The keys I've noticed were being intercepted is mainly just cmd+tab. As you can see I can cmd+tab before starting fig, but after it's enabled I can't cmd+tab out of the terminal or vscode.

https://user-images.githubusercontent.com/7784127/133752613-cc9dcd40-18c7-490b-932d-30dedd3a2d66.mp4

PS: I have `autocomplete.onlyShowOnTab` set to `true`

**Comment by mschrage** [comment]
Thanks for the video! This confirms my suspicion of what was going on. This issue will be fixed in the next release (v1.0.49) :)

**Comment by GuilleDF** [comment]
@mschrage That's great to know, thanks! 😃

**Comment by fwesss** [comment]
@GuilleDF 1.0.49 is out 🙂
Can you run `fig update` and let me know if that fixes this for you?

**Comment by GuilleDF** [comment]
@mschrage It works, thanks! 😄
